index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <title>Home</title>
+</head>
+<body id="body">
+    <!-- first page for home -->
+    <header class="header" id="home">
+        <nav class="nav-top-desktop">
+            <ul>
+                <li><i class="fa-brands fa-facebook-f"></i></li>
+                <li><i class="fa-brands fa-twitter"></i></li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </nav>
+        <nav>
+            <div class="nav-top-mobile">
+                <button id="wrapper"><i class="fa-sharp fa-solid fa-bars"></i></button>
```

Comment:
- [ ] I think the Hamburger menu it still show while scrolling but that is not very good user experience so kindly position it to just be in the nav or if you want to make your nav sticky you can give a background color to the nav so it still in the page while scrolling ✨⭐♻🚀💯

- Your design
![image](https://user-images.githubusercontent.com/58553711/219303755-a253e07b-9ca0-48f5-848b-9f4456bd91b9.png)